app/workers/metrics/record_daily_notifications_worker.rb

```diff
@@ -0,0 +1,34 @@
+module Metrics
+  class RecordDailyNotificationsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    EVENT_TITLES = %w[
+      welcome_notification_set_up_profile
+      welcome_notification_welcome_thread
+      welcome_notification_customize_feed
+      welcome_notification_twitter_connect
+      welcome_notification_github_connect
+      welcome_notification_customize_experience
+      welcome_notification_discuss_and_ask
+      welcome_notification_download_app
+      welcome_notification_ask_question
+      welcome_notification_start_discussion
+    ].freeze
```

Comment:
This isn't the most extensible way to do this, but since there's a lot of [other refactoring](https://github.com/thepracticaldev/dev.to/pull/9231) going on in the welcome notifications/broadcasts portion of the codebase, I decided to just go with what works for now. Once the other work is done, we'll be able to make some of this work more "Forem-friendly", and can revisit this.